distribution/src/resources/drill-config.sh

```diff
@@ -392,5 +392,7 @@ export DRILL_HOME
 export DRILL_CONF_DIR
 export DRILL_LOG_DIR
 export CP
-export JAVA_HOME
+# DRILL-4870: Don't export JAVA_HOME. Java can find it just fine from the java
+# command. If we attempt to work out out, we do so incorrectly for the Mac.
```

Comment:
Yikes! Fixed the typo.

Changed the reference to JAVA_HOME to a reference relative to java, which we must already know.